e2e/bitrise.yml

```diff
@@ -0,0 +1,201 @@
+    - SAMPLE_APP_BRANCH: master
+    - OUTPUT_TOOL: xcodebuild
+    - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
+    - BITRISE_SCHEME: ios-simple-objc
+    - SIMULATOR_DEVICE: iPhone 8 Plus
+    - SIMULATOR_OS_VERSION: "latest"
+    - SIMULATOR_PLATFORM: iOS Simulator
+    after_run:
+    - _run
+    - _check_outputs
+    - _check_exported_artifacts
+
+  test_objc_xcpretty:
+    envs:
+    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
+    - SAMPLE_APP_BRANCH: master
+    - OUTPUT_TOOL: xcpretty
+    - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
+    - BITRISE_SCHEME: ios-simple-objc
+    - SIMULATOR_DEVICE: iPhone 8 Plus
+    - SIMULATOR_OS_VERSION: "latest"
+    - SIMULATOR_PLATFORM: iOS Simulator
+    after_run:
+    - _run
+    - _check_outputs
+    - _check_exported_artifacts
+    - _check_xcpretty_html_report
+
+  test_failing_ui_test:
+    steps:
+    - script:
+        title: Start a failing workflow, wrapped in a script.
+        inputs:
+        - content: |-
+            #!/bin/env bash
+            set -ex
+            bitrise run --config=./e2e/bitrise.yml utility_objc_failing_ui_test
+            if [ $? -ne 1 ] ; then
+              echo "Workflow was excepted to fail, exit code not 1."
+              exit 1
+            fi
+
+  utility_objc_failing_ui_test:
+    envs:
+    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
+    - SAMPLE_APP_BRANCH: failing-ui-test
+    - OUTPUT_TOOL: xcodebuild
+    - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
+    - BITRISE_SCHEME: ios-simple-objc
+    - SIMULATOR_DEVICE: iPhone 8 Plus
+    - SIMULATOR_OS_VERSION: "latest"
+    - SIMULATOR_PLATFORM: iOS Simulator
+    after_run:
+    - _run
+    - _check_outputs
+    - _check_exported_artifacts
+
+  test_tvos:
+    envs:
+    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-tvos-swift.git
+    - SAMPLE_APP_BRANCH: master
+    - OUTPUT_TOOL: xcodebuild
+    - BITRISE_PROJECT_PATH: NPO Live.xcworkspace
+    - BITRISE_SCHEME: NPO Live
+    - SIMULATOR_DEVICE: Apple TV
+    - SIMULATOR_OS_VERSION: "latest"
+    - SIMULATOR_PLATFORM: tvOS Simulator
+    after_run:
+    - _run
+    - _check_outputs
+
+  test_parallel:
+    envs:
+    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-swift-project-with-parallel-ui-test.git
+    - SAMPLE_APP_BRANCH: master
+    - OUTPUT_TOOL: xcodebuild
+    - BITRISE_PROJECT_PATH: BullsEye.xcodeproj
+    - BITRISE_SCHEME: BullsEye
+    - SIMULATOR_DEVICE: iPhone 8 Plus
+    - SIMULATOR_OS_VERSION: "latest"
+    - SIMULATOR_PLATFORM: iOS Simulator
+    after_run:
+    - _run
+    - _check_outputs
+
+  _run:
+    steps:
+    - script:
+        inputs:
+        - content: |-
+            #!/bin/env bash
+            set -ex
+            rm -rf ./_tmp
+    - change-workdir:
```

Comment:
change-workdir not need: https://github.com/bitrise-steplib/steps-xcode-archive/pull/230/files#diff-6505b061ae185d6fd80374148e0e39774b48ebf7aae5ccbfe2931428da50f814R133-R170

Reply:
Updated